tutorials/object_detection.ipynb

```diff
@@ -0,0 +1,269 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3c5b2a6f",
+   "metadata": {},
+   "source": [
+    "### Establish Connection With Eva DB"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "62e83a0b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.insert(0,'..')\n",
+    "from src.server.db_api import connect"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "b78a43e3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import nest_asyncio\n",
+    "# nest_asyncio.apply()\n",
+    "connection = connect(host = '0.0.0.0', port = 5432) # hostname, port of the server where EVADB is running"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "1bc99e88",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cursor = connection.cursor()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6d64d3b6",
+   "metadata": {},
+   "source": [
+    "### Upload the survellience video to analyse"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "130b8561",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cursor.execute('UPLOAD INFILE \"../data/ua_detrac/ua_detrac.mp4\" PATH \"video.mp4\";')\n",
+    "response = cursor.fetch_one()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6450f505",
+   "metadata": {},
+   "source": [
+    "### Visualize Video"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "bef7026f",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "19baca4d84dc4c58ae918e0ba18414bb",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Video(value=b'\\x00\\x00\\x00 ftypisom\\x00\\x00\\x02\\x00isomiso2avc1mp41\\x00\\x00\\x00\\x08free\\x00\\x19K[mdat\\x00\\x00\\…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from ipywidgets import Video\n",
+    "Video.from_file(\"../data/ua_detrac/ua_detrac.mp4\", embed=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "686adeb5",
+   "metadata": {},
+   "source": [
+    "### Load video into EVA"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "b18cfe2a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cursor.execute('LOAD DATA INFILE \"test_video.mp4\" INTO MyVideo;')\n",
+    "response = cursor.fetch_one()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "740b38cb",
+   "metadata": {},
+   "source": [
+    "### Optional - Register FasterRCNN (object detection) model into EVA "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "e83e5a44",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cursor.execute(\"\"\"CREATE UDF IF NOT EXISTS FastRCNNObjectDetector \n",
```

Comment:
Would it be possible to elaborate on the `INPUT` and the `OUTPUT` in this query? Specifically, the purpose of the constructs `frame`, `labels`, and '`bboxes`, and how they affect the construction of the UDF. We could provide a brief explanation as a markdown cell in the notebook itself.

Reply:
Done. We should add similar information for other queries as well. Probably, will need help with that.